### PR TITLE
to remove large space after first comment

### PR DESCRIPTION
**Pull request by Ivanov-Anton**
Screenshot 2020-10-12 at 16 06 17

### CODE REVIEW

File: app/assets/stylesheets/active_admin_sidebar.scss [hunk 1]
```diff
@@ -85,3 +85,11 @@ body.active_admin.index #active_admin_content.with_sidebar.collapsible_sidebar {
     margin-right: 0;
   }
 }
+
+.active_admin_comment:not(#new_active_admin_comment) {
+  display: flex;
+}
+
+.active_admin_comment_body {
+  margin-left: 20px!important;
```
Comment (Fivell) [inline app/assets/stylesheets/active_admin_sidebar.scss:94]: please to not use important

Comment (Fivell) [reply app/assets/stylesheets/active_admin_sidebar.scss:94]: ```body.with_sidebar .comments . active_admin_comment  .active_admin_comment_body```

Comment (Ivanov-Anton) [reply app/assets/stylesheets/active_admin_sidebar.scss:94]: @fivell, This ready-made solution?

Comment (Fivell) [reply app/assets/stylesheets/active_admin_sidebar.scss:94]: @Ivanov-Anton no idea

**Comment by workgena** [comment]
@Ivanov-Anton, please, attach screenshot with current layout before patch

I couldn't find an issue with "large space after first comment".

browser Google Chrome ?

**Comment by Ivanov-Anton** [comment]
> @Ivanov-Anton, please, attach screenshot with current layout before patch
> 
> I couldn't find an issue with "large space after first comment".
> 
> browser Google Chrome ?

okay wait please

**Comment by Ivanov-Anton** [comment]
@workgena 
Screenshot 2020-10-27 at 16 37 33

**Comment by workgena** [comment]
Oh, I see. Looks pretty bad.

Can you please test it with default ActiveAdmin theme?

This fix might be related only to activeadmin-plugins/active_admin_theme

**Comment by Ivanov-Anton** [comment]
> Oh, I see. Looks pretty bad.
> 
> Can you please test it with default ActiveAdmin theme? And confirm/refute issue.
> 
> This fix might be related only to [activeadmin-plugins/active_admin_theme](https://github.com/activeadmin-plugins/active_admin_theme)

moment

**Comment by Ivanov-Anton** [comment]
Screenshot 2020-10-27 at 16 56 13

**Comment by workgena** [comment]
Interesting 🤔 

Please show your Gemfile. I'll try to take a closer look at this problem.

**Comment by Ivanov-Anton** [comment]
Screenshot 2020-10-27 at 16 57 23

@workgena

**Comment by workgena** [comment]
I suspect the fix might be 

```css
.active_admin_comment_body {
  overflow:hidden;
}
```

**Comment by Ivanov-Anton** [comment]
> ```css
> overflow:hidden;
> ```

just tried in Google Chrome dev tools, not work :(

**Comment by Ivanov-Anton** [comment]
> I suspect the fix might be (just a thought)
> 
> ```css
> .active_admin_comment_body {
>   overflow:hidden;
> }
> ```

**Comment by Ivanov-Anton** [comment]
@workgena  do you have any idea?

**Comment by workgena** [comment]
The height of "space" depends on sidebar height. I have experience with such behaviour, but don't remember the solution, something with "height: 100%". Need time to investigate.

**Comment by Ivanov-Anton** [comment]
> The height of "space" depends on sidebar height. I have experience with such behaviour, but don't remember the solution, something with "height: 100%". Need time to investigate. It is because of "display: float"

thank you very much for your time

**Comment by Ivanov-Anton** [comment]
maybe tomorrow you will find 20 minutes

**Comment by workgena** [comment]
Here is the solution:

```css
.comments .active_admin_comment {
  overflow: auto;
}
```

If this does not works for your case, then change "auto" to "hidden"

**Comment by Ivanov-Anton** [comment]
> Here is the solution:
> 
> ```css
> .comments .active_admin_comment {
>   overflow: auto;
> }
> ```
> 
> If this does not works for your case, then change "auto" to "hidden"

checking

**Comment by workgena** [comment]
I've tested only in Firefox/Chrome (Ubuntu Linux).

And one request :wink: please check this in Edge(latest) and Safari. And in mobile Safari/Chrome.

**Comment by Ivanov-Anton** [comment]
> I've tested only in Firefox/Chrome (Ubuntu Linux).
> 
> And one request 😉 please check this in Edge(latest), Safari and mobile Safari/Chrome.

Good job @workgena !

**Comment by Ivanov-Anton** [comment]
@workgena  do you think this patch should be put in this gem or in active_admin_theme?

**Comment by workgena** [comment]
> @workgena do you think this patch should be put in this gem or in active_admin_theme?

It should be here in **active_admin_sidebar**

File: app/assets/stylesheets/active_admin_sidebar.scss [hunk 2]
```diff
@@ -85,3 +85,7 @@ body.active_admin.index #active_admin_content.with_sidebar.collapsible_sidebar {
     margin-right: 0;
   }
 }
+
+.comments .active_admin_comment {
+  overflow: auto;
```
Comment (Fivell) [inline app/assets/stylesheets/active_admin_sidebar.scss:90]: .with_sidebar .comments .active_admin_comment  ?? @workgena

Comment (workgena) [reply app/assets/stylesheets/active_admin_sidebar.scss:90]: Yes 💯

**Comment by workgena** [comment]
@Ivanov-Anton this should help with failing CI:

* https://github.com/activeadmin-plugins/active_admin_scoped_collection_actions/pull/34/files
* https://github.com/activeadmin-plugins/active_admin_scoped_collection_actions/pull/37/files

**Review by gigorok — APPROVED** [review]

**Pull request merged**